### DISCUSSION TITLE
Adding model path/ specification in .h5 files

### DISCUSSION THREAD

**Discussion by NeuTTH** [talmolab/sleap#1277] [Ideas]
There might be scenarios when inference results are mixed together when they are trained by different models/ configurations. It will be nice if there is an option to include the model specification/ path in the h5 file so people can check which model was used.

**Comment by roomrys** [comment]
Hi @NeuTTH,

We have a bit of insight into which model is used from within the `provenance` dataset in the h5:
https://github.com/talmolab/sleap/blob/b5aab0edfbcfddce82e8ee02d9bd383604b9de93/sleap/info/write_tracking_h5.py#L354-L370

However, this will only include the latest model information (the last model that was used to run inference). So unless you are predicting on all frames, this info will not help decipher which predictions came from which model.

Thanks for the idea!
Liezl
